packages/core/src/content/content-client.ts

```diff
@@ -122,7 +122,7 @@ export class ContentClient {
     debug.content('Getting locale for id: %s', id);
 
     const response = await this.get<LocaleQueryResponse>(GET_LOCALE_QUERY, { id });
-    return response.locale;
+    return response.locale ? response?.locale?.system : null;
```

Comment:
Very minor thing for simplification ;)
```suggestion
    return response.locale?.system || null;
```